scm/src/gmtb_scm.F90

```diff
@@ -38,11 +38,19 @@ subroutine gmtb_scm_main_sub()
   logical                           :: in_spinup
 
   call get_config_nml(scm_state)
-
-  call get_case_init(scm_state, scm_input)
-
+  
```

Comment:
I wonder if, at some point, we should switch to strings instead of cryptic integers.

Reply:
Sure, one could do that. I think that I subconsciously avoid using strings as much as possible in Fortran due to them being difficult to handle in the past. In this particular case, though, I'm hoping to drop the need for the input_type altogether eventually when ONLY the DEPHY format is supported.